### PR TITLE
add documentation to navbar and animations classes [#159932797]

### PR DESCRIPTION
**Pull request by ajzoell**
Note: anything added to the "documentation" folder is auto-generated. No need to check these. All changes to program occurred in "src" folder.

Any changes to classes other than navbar and animations have been reverted. They should match broker.

If you want to check out the compodoc page launched on localhost:

1.) If you have not already, run npm install -g @compodoc/compodoc to install compodoc globally.
2.) Navigate to the folder containing the documentation folder for this repository.
3.) Run compodoc -s -o to automatically serve and open the compodoc documentation on localhost:8080.

### CODE REVIEW

File: src/app/guards/auth.guard.ts [hunk 1]
```diff
@@ -10,12 +10,12 @@ import { AuthService } from '../services/auth.service';
...
  * if they are logged in it returns true and they gain access to Selection and Forecast
- * if a user tries to access a page that has canActivate on it and they are not logged in
- * they will be redirected to '' (Homepage)
+ * if a user tries to access a page that has canActivate on it and they are not logged in 
+ * they will be redirected to '' (Homepage).
  */
-
 export class AuthGuard implements CanActivate {
-  // add the service we need
+
```
Comment (Hoagithor) [inline src/app/guards/auth.guard.ts]: unnecessary whitespace

File: src/app/guards/auth.guard.ts [hunk 2]
```diff
@@ -10,12 +10,12 @@ import { AuthService } from '../services/auth.service';
 /**
  * boolean function that is called to check to see if user is logged in or not
  * if they are logged in it returns true and they gain access to Selection and Forecast
- * if a user tries to access a page that has canActivate on it and they are not logged in
- * they will be redirected to '' (Homepage)
+ * if a user tries to access a page that has canActivate on it and they are not logged in 
+ * they will be redirected to '' (Homepage).
```
Comment (Hoagithor) [inline src/app/guards/auth.guard.ts]: Inconsistent capitalization/punctuation, should be for each sentence (consistent with other doc)

File: src/app/services/auth.service.ts [hunk 3]
```diff
@@ -9,14 +9,15 @@ export class AuthService {
         graphScopes: ['user.read'],
         authority: 'https://login.microsoftonline.com/project3auth.onmicrosoft.com',
         extraQueryParameter: 'domain_hint=project3auth.onmicrosoft.com'
-    };
+    };
```
Comment (Hoagithor) [inline src/app/services/auth.service.ts]: uneeded space at end of line (since the review is already open)

File: src/app/modules/routing/routing.module.ts [hunk 4]
```diff
@@ -8,7 +8,7 @@ import { SelectionComponent } from '@revaturecloud/selection';
 import { AuthGuard } from '../../guards/auth.guard';
 
 /**
- * Routes for the application
+ * routes for the application
  * both have canActivate constraint which requires user to be logged in to access them
```
Comment (Hoagithor) [inline src/app/modules/routing/routing.module.ts]: Inconsistent capitalization/punctuation with other Doc

File: src/app/guards/auth.guard.ts [hunk 5]
```diff
@@ -10,12 +10,12 @@ import { AuthService } from '../services/auth.service';
...
  */
-
 export class AuthGuard implements CanActivate {
-  // add the service we need
+  
+  // add the service we need.
```
Comment (Hoagithor) [inline src/app/guards/auth.guard.ts]: Inconsistent capitalization with other doc

File: src/app/guards/auth.guard.ts (hunk 1 above)
Comment (ajzoell) [reply src/app/guards/auth.guard.ts]: We can't alter this one. It's not part of this user story.

File: src/app/guards/auth.guard.ts (hunk 2 above)
Comment (ajzoell) [reply src/app/guards/auth.guard.ts]: Not part of this user story. We needed to revert it.

File: src/app/services/auth.service.ts (hunk 3 above)
Comment (ajzoell) [reply src/app/services/auth.service.ts]: Not part of this user story.

File: src/app/modules/routing/routing.module.ts (hunk 4 above)
Comment (ajzoell) [reply src/app/modules/routing/routing.module.ts]: We had to revert this. It's not part of this user story.

**Comment by ajzoell** [comment]
Many of the changes requested didn't match broker branch. We needed to revert some changes made due to changes made outside of our user story...
